containers/scripts/deploy.sh

```diff
@@ -80,7 +80,8 @@ prepare_deploy() {
         return 4
     fi
 
-    clean
+    # Bring compose down if `FORCE` is true.
```

Comment:
AFAIK, this is not what happens. `--force` only makes it possible to continue even when the containers are running.  If they are stopped, the compose will be always brought down.

Reply:
@siteshwar Thanks for the fix!  However, this comment is still not correct.